src/Handlers.cpp.Rt

```diff
@@ -2004,7 +2062,33 @@ class acSyntheticTurbulence: public Action {
 			}
 			output("Setting one-wave spectrum with length=%lg\n", 1/WN); 
 			solver->lattice->ST.setOneWave(WN);
-		} else {
+		}/* else if (spec == "Free stream") {
```

Comment:
Let's get rid of this commented out section. I think "Intensity" can be added directly to the "Von Karman" spectrum as an option - but that's a matter for another pull request.